### PR TITLE
Refactor payment pages + add reconciliation information as JSON

### PR DESCRIPTION
**Pull request by dacspiir**

### CODE REVIEW

File: Web/Controllers/PaymentV2Controller.cs [hunk 1]
```diff
@@ -119,6 +119,10 @@ public async Task<IActionResult> Payments()
         [FromRoute] string paymentAuthorizationId)
     {
         if (_environment.IsProduction()) return NotFound();
+
+        return NotFound();
+        
+        /*
         try
```
Comment (adzspiir) [inline Web/Controllers/PaymentV2Controller.cs:126]: should we keep it commented-out? should we just remove it?

File: Web/Controllers/PaymentV2Controller.cs [hunk 2]
```diff
@@ -152,23 +157,7 @@ public IActionResult PaymentAuthorizationCallback([FromQuery] string authorizati
...
-                var payment = await _aiiaService.GetInboundPayment(User, accountId, paymentId);
-                return View("ObjectDetailsView",  new ObjectDetailsViewModel("Inbound payment V2", payment, payment.Id));
-            }
-            catch (AiiaClientException)
-            {
-                return View("ObjectDetailsView");
-            }
-        }
+        var payment = await _aiiaService.GetOutboundPaymentV2(User, accountId, paymentId);
+        return View("ViewPaymentV2", new ViewPaymentV2ViewModel(payment));
```
Comment (adzspiir) [inline Web/Controllers/PaymentV2Controller.cs:161]: how do we know it's an outbound payment? do we have to do the same dance as before, i.e. try GetOutBoundPayment, if that fails, try GetInboundPayment?

File: Web/Models/ViewPaymentV1ViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        Payment = payment;
+        PaymentType = paymentType;
+        Reconciliation = reconciliation;
+
+    }
+
+    public object Payment { get; set; }
+    public PaymentType PaymentType { get; set; }
+    public PayerTokenModel PayerToken { get; set; }
```
Comment (adzspiir) [inline Web/Models/ViewPaymentV1ViewModel.cs:18]: could PayerToken be one of the arguments in the constructor? It is easy to forget to update it after creation IMO

Comment (adzspiir) [reply Web/Models/ViewPaymentV1ViewModel.cs:18]: I guess the payer token is only relevant for the accept payments, so i understand if you dont want to have it by default

File: Web/Controllers/PaymentV2Controller.cs (hunk 1 above)
Comment (dacspiir) [reply Web/Controllers/PaymentV2Controller.cs:126]: I think I just forgot about it 😅 
I'll add the feature back - thanks!

**Review by adzspiir — APPROVED** [review]

**Pull request merged**